datumaro/cli/commands/merge.py

```diff
@@ -3,16 +3,16 @@
 # SPDX-License-Identifier: MIT
 
 import argparse
+from collections import OrderedDict
 import json
 import logging as log
 import os.path as osp
```

Comment:
I'm not sure I understand the order it produces. What are the rules for `import` and `from import`?